app/scripts/modules/core/postcss.config.js

```diff
@@ -1,11 +0,0 @@
-const colorMap = require('@spinnaker/styleguide/src/colorMap');
-
-module.exports = {
-  plugins: {
-    autoprefixer: {},
-    'postcss-colorfix': {
-      colors: colorMap,
-    },
-    'postcss-nested': {},
-  },
-};
```

Comment:
Do we have it somewhere else?

Reply:
Yep, this is handled in [scripts](https://github.com/spinnaker/deck/blob/master/packages/scripts/config/rollup.config.base.js#L46-L55)